src/components/VulnerabilitiesBlock/SubscoresBlock/SubscoresBlock.style.tsx

```diff
@@ -0,0 +1,33 @@
+import styled, { css } from 'styled-components';
+import { GREY_0, COLOR_MAP } from 'common/colors';
+
+export const Row = styled.div<{ grayRow?: boolean }>`
+  background-color: ${({ grayRow }) => grayRow && GREY_0};
+  display: flex;
+`;
+
+const CellStyles = css`
+  display: flex;
+  align-items: center;
+  width: 100%;
+`;
+
+export const ScoreNameCell = styled.div`
+  ${CellStyles};
+  color: ${COLOR_MAP.GRAY_BODY_COPY};
+  font-size: 0.875rem;
+  padding: 10px 12px;
+  line-height: 1.25;
+`;
+
+export const LevelCell = styled.div<{ color: string }>`
+  ${CellStyles};
+  background-color: ${({ color }) => color};
+  color: white;
+  max-width: 80px;
+  padding: 12px 8px;
+  font-weight: 500;
+  font-size: 0.7rem;
+  text-transform: uppercase;
```

Comment:
This is a bit weird, VoiceOver spells some levels intead of reading them. Can you add the level text in lowercase as `aria-label` for the component?

Reply:
It's probably because it's shown in uppercase